pages/Technical Documentation/Data/Contributing to the EarthCODE Catalog.md

```diff
@@ -242,16 +252,383 @@ Alternatively, if you have access to a **reliable cloud storage service**—such
 
 If you choose to use the ESA PRR the generated STAC Items have to be sent via `POST` requests to the [PRR Registration Gateway](https://eoresults.esa.int/reg-api/docs#/) for registration using the previously requested collection.
 
-The only requirement is that __other users and the STAC browser must be able to find and read your repository!__
+The only requirement is that **other users and the STAC browser must be able to find and read your repository!**
 
 :::info  
 In the next step where you will be uploading metadata to the Open Science Catalog, EarthCODE administrators will review your Item Catalog and assist you with any necessary adjustments.  
-:::  
+:::
+
+## Step 3: Creating a Product Entry in the Open Science Catalog
+
+After completing steps 1 and 2 to prepare your data package, you're now ready to add a product entry to the Open Science Catalog (OSC).
+
+This involves creating a new collection folder under the `products` directory in the [`open-science-catalog-metadata-staging`](https://github.com/ESA-EarthCODE/open-science-catalog-metadata-staging) repository. The process is straightforward, especially if you're already familiar with Git.
+
+Your `collection.json` file should include general metadata and links to related resources:
+
+- A link to your Item Collection
+- Links to relevant metadata collections (e.g., Variables, Missions, Projects, and Themes)
+- Other useful links, such as documentation, DOIs, websites, etc.
+
+Refer to the diagram below for a detailed overview of the required fields for OSC products:
+
+![metadata-stac](https://github.com/EOEPCA/open-science-catalog-metadata/assets/120453810/71b8e8a7-9a86-491b-ae54-1fb4de9ccf32)
```

Comment:
Maybe add a diagram showing the OSC structure along the lines of this:

![OSC-Structure](https://github.com/user-attachments/assets/93ebb5b0-7aea-4907-9779-a3e4216e053e)